homeassistant/components/sensibo/diagnostics.py

```diff
@@ -0,0 +1,21 @@
+"""Diagnostics support for Sensibo."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import SensiboDataUpdateCoordinator
+
+TO_REDACT: dict[str, Any] = {}
```

Comment:
TO_REDACT is empty, so there is not point in using async_redact_data.

Reply:
If you decide to keep it, it should be a set:
```suggestion
TO_REDACT: set[str] = set()
```

But I would just remove it completely.